### PR TITLE
Complete design change and change of controls using  dash-bootstrap

### PR DESCRIPTION
**Pull request by ruchimotwaniBC**

### CODE REVIEW

File: doors_dashboards/components/scatterplot.py [hunk 1]
```diff
@@ -1,136 +1,1104 @@
...
-                        style={
-                            'height': '40vh'
-                        },
+            fig.update_layout(
+                font=dict(family=FONT_FAMILY, size=18, color=FONT_COLOR),
+                plot_bgcolor="rgb(0,0,0,0)",
+                paper_bgcolor='rgba(0,0,0,0)',
+                legend=dict(
+                    title=dict(
+                        text='Sampling Depth [M]'.title(),
```
Comment (TonioF) [inline doors_dashboards/components/scatterplot.py:376]:
```suggestion
                        text=levels[-1].title(),
```

File: doors_dashboards/components/scatterplot.py [hunk 2]
```diff
@@ -1,136 +1,1104 @@
...
-                id=SCATTER_PLOT_LINE_ID,
-                figure=lineplot_fig,
-                style={
-                    'height': '40vh'
-                },
+                )
             )
+            fig.update_traces(marker_size=10)
+            fig.update_xaxes(title_text=variables[0].title())
+            fig.update_yaxes(title_text=variables[1].title())
```
Comment (TonioF) [inline doors_dashboards/components/scatterplot.py:388]:
```suggestion
            fig.update_xaxes(title_text=x_variable.title())
            fig.update_yaxes(title_text=y_variable.title())
```

**Review by TonioF — CHANGES_REQUESTED** [review]
The module dashboard.py has become unnecessary. Please remove it, or better, replace its content with that of dashboard_bootstrap.py and delete that one. Also, have a look at #26 and my other comments.

**Comment by ruchimotwaniBC** [comment]
> The module dashboard.py has become unnecessary. Please remove it, or better, replace its content with that of dashboard_bootstrap.py and delete that one. 

I have already done it in my new branch created from this. Shall I do it in this branch and commit it??

**Comment by TonioF** [comment]
> > The module dashboard.py has become unnecessary. Please remove it, or better, replace its content with that of dashboard_bootstrap.py and delete that one.
> 
> I have already done it in my new branch created from this. Shall I do it in this branch and commit it??

Yes, please.

**Review by TonioF — APPROVED** [review]

**Pull request merged**